### ISSUE TITLE
On carrel pages ADD carrel credit/creator info & preferred citation

### ISSUE THREAD

**Issue by nkmeyers** [ericleasemorgan/reader#49]
On carrel pages ADD carrel credit/creator info & citation and preferred citation, potentially as footer: e.g.
Meyers, N. (2020, March 26) “Distant Reading the US Academic Library Response to COVID19” URL=https://carrels.distantreader.org/library/wolff/index.htm 

related to #32

**Comment by ericleasemorgan** [comment]
Yes, the linked repository is expected to provide this sort of functionality: https://github.com/ericleasemorgan/reader-carrels
